search/linear_search.cpp

```diff
@@ -63,30 +67,42 @@ static void tests() {
  * @brief Main function
  * @returns 0 on exit
  */
-int main() {
+int main()
+{
     int mode = 0;
 
     std::cout << "Choose mode\n";
     std::cout << "Self-test mode (1), interactive mode (2): ";
 
     std::cin >> mode;
 
-    if (mode == 2) {
+    if (mode == 2)
+    {
         int size = 0;
-        std::cout << "\nEnter the size of the array: ";
+        std::cout << "\nEnter the size of the array [in range 1-30 ]: ";
         std::cin >> size;
 
-        while ((size <= 1) || (size >= 30)) {
-            std::cout << "Size cannot be less than zero. Please choose another value: ";
-            std::cin >> size;
+        while (size <= 0 || size > 30)
+        {
+            if (size <= 0)
+            {
+                std::cout << "Size cannot be less than zero. Please choose another value: ";
+                std::cin >> size;
+            }
+            if (size > 30)
+            {
+                std::cout << "Size entered more than 30. Please choose a lower value: ";
+                std::cin >> size;
+            }
```

Comment:
To make it shorter, we could do something like this. What do you think? 🙂
```suggestion
        while (size <= 0 || size > 30){
            std::cout << "Size can only be 1-30. Please choose another value: ";
            std::cin >> size;
        }
```

Reply:
Yes this looks fine, thankyou 🙂
Should I commit this suggestion?

Reply:
Looks good guys.